README.md

```diff
@@ -14,65 +14,81 @@ MIT License
 
 ## Getting started
 
-  1. Add `commanded_audit_middleware` to your list of dependencies in `mix.exs`:
-
-      ```elixir
-      def deps do
```

Comment:
Note: Most of the README.md changes are  automated reformattings. The only substantive change is in section 2, where I  describe how to optionally use JSONB.